charts/ark-solr/values.yaml

```diff
@@ -0,0 +1,64 @@
+# Default values for solr-helm-charts.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+imagePullSecrets: []
+
+#Common Parameters
+nameOverride: ""
+
+fullnameOverride: ""
+
+#Comman Labels to add to all solr resources
+commonLabels: {}
+port: 8983
+## deployment options
+# general: Minikube 
+# eks: Amazon Elastic Kubernetes Service
+deployment: ""
+image:
+  repository: 345280441424.dkr.ecr.ap-south-1.amazonaws.com/arkcase-solr
+  pullPolicy: Always 
```

Comment:
@naveen-armedia Why not `IfNotPresent`? I think `Always` is only useful for floating tags such as `latest`, because the image they point to might change with time. But for fixed tags, that's wasting bandwidth and slowing down deployments.

@JonHolman @drivera-armedia What do you guys think?

Reply:
@fabricetriboix  Yes we can apply `IfNotPresent` do you want me change it?

Reply:
@naveen-armedia Yes, I think it makes sense to use `IfNotPresent` unless we use floating tags (which is not the case here). Please go ahead and make the change.